test/humanizer.js

```diff
@@ -124,6 +124,20 @@ describe('humanizer', function () {
     assert.equal(h(2838550, { largest: 3 }), '47 minutes, 19 seconds')
   })
 
+  it('can do rounding with the "maxDecimalPoint" option', function () {
```

Comment:
1. Could you test the behavior when `maxDecimalPoints === 0`?
1. `"maxDecimalPoint"` → `"maxDecimalPoints"`